compiler/rustc_builtin_macros/src/deriving/generic/mod.rs

```diff
@@ -222,14 +222,21 @@ pub struct MethodDef<'a> {
 
     pub attributes: ast::AttrVec,
 
-    /// Can we combine fieldless variants for enums into a single match arm?
-    /// If true, indicates that the trait operation uses the enum tag in some
-    /// way.
-    pub unify_fieldless_variants: bool,
+    pub unify_fieldless_variants: UnifyFieldlessVariants,
```

Comment:
I think this field should be renamed to something like `field_variants_strategy`.